### DISCUSSION TITLE
How to use "resx" in a Library?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [dotnet/maui#23673] [Q&A]
Hello,

I migrated my shared Xamarin project to .NET MAUI Library. I need to use translations that are stored there. They need to be accessible from this library and from the MAUI app project that references this library. 

How can I configure my `csproj` to generate strongly typed properties for the `resx` file? I couldn't find any information on that. Tutorial only says to put this file in MAUI app project. However, then it wouldn't be accessible by my modules inside .NET MAUI Library (migrated Xamarin shared project)?

So far I figured out that I have to rename the previously generated file `Strings.Designer.cs` to `Strings.cs` and replace `internal` with `public`. However, this is just a workaround, because the new translations are not auto-generated.

**Comment by gabsamples6** [comment]
We use Rex files in a dedicated project. We have a white label app and a build time we copy a client specific Rex file to this project and that is it .in the after build event in visual studio you could copy it.0r in the azure pipeline u can do it .
